.github/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
 
 ## 0.8.0
+1. [FBW] Improved speed dependent roll law model - @IbrahimK42 (IbrahimK42)
```

Comment:
**nitpick**: Changelog entries should go to the bottom of the list 😀.

Reply:
Ups, it was my first PR ever :) Will pay attention next time